### DISCUSSION TITLE
Only webhook / user filter option? (Suggestion)

### DISCUSSION THREAD

**Discussion by biast12** [Forcellrus/Discord-Auto-Publisher#21] [Ideas]
I'm making this Suggestion because we got a server with a git webhook posting when we push things and sometimes do we also make small comments in that channel that we don't want to get published to other servers.

this option could maybe work as a blacklist / whitelist option with commands, like `?blacklist <UserID / WebhookID>` or `?whitelist <UserID / WebhookID>`

tho one thing that would be nice if it could be done for that specific channel, so we still can post updates in another channel without it being filtered off

**Comment by acehox** [comment]
Hey, such feature is planned for the future.
